UIInfoSuite2/i18n/es.json

```diff
@@ -33,5 +33,11 @@
   "ShowRainyDay": "Show rainy days", // TODO
   "ShowHarvestPricesInShop": "Mostrar los precios de cosechas de la tienda",
   "ShowWhenNewRecipesAreAvailable": "Mostrar cuando hay nuevas recetas disponibles",
-  "ShowToolUpgradeStatus": "Mostrar el estado de actualización de la herramienta"
+  "ShowToolUpgradeStatus": "Mostrar el estado de actualización de la herramienta",
+  "LuckStatus1": "¡Tienes suerte!",
+  "LuckStatus2": "Te sientes con suerte... aunque no demasiado",
```

Comment:
It would be good to add a // TODO notation to LuckStatus2-LuckStatus5, as they are not exact translations and should be improved. When we have a few pending translations, I will add this repo to the [mod translation request ](https://github.com/StardewModders/mod-translations/issues) page.